rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -113,8 +113,8 @@ <h1 class="col-sm-10 col-md-11">{{ record.title }}</h1>
             <li>
               {{ identifier.value }}
               <small class="badge badge-secondary text-uppercase">{{ identifier.type }}</small>
-              {% endfor %}
             </li>
+            {% endfor %}
```

Comment:
Can you add this fix to the commit message?

Reply:
ok